pkg/ddevapp/global_dotddev_assets/commands/web/craft

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+#ddev-generated
+## Description: Run Craft CMS command inside the web container
+## Usage: craft [flags] [args]
+## Example: "ddev craft db/backup" or "craft db/backup ./my-backups" (see https://craftcms.com/docs/4.x/console-commands.html)
+## ProjectTypes: php
```

Comment:
This is the one thing I thought of after we talked, there's not yet a craft project type. But using it with project type php is a pretty decent solution probably. May annoy some people using the "php" project type on other CMSs that "craft" shows up as a command.

Reply:
Yes, agreed that it's not ideal. I think it might be better to wait for @angrybrad to submit his PR, which I assume will include this command anyway.

Reply:
I think it's OK though.